docs/platforms/flutter/integrations/slow-and-frozen-frames-instrumentation.mdx

```diff
@@ -32,15 +33,15 @@ This type of instrumentation is automatically enabled. There is no need for furt
 
 ### Disabling the Instrumentation
 
-Set `enableAutoPerformanceTracing` to `false` in the options to disable the instrumentation.
+Set `enableFramesTracking` to `false` in the options to disable the instrumentation.
```

Comment:
it's great that this is now decoupled from routing instrumentation and that users can deactivate it separately from overall tracing 👍